assets/js/components/Work/Tabs/About/CoreMetadata.jsx

```diff
@@ -1,20 +1,23 @@
 import React from "react";
 import PropTypes from "prop-types";
-import UITagNotYetSupported from "../../../UI/TagNotYetSupported";
 import UIInput from "../../../UI/Form/Input";
 import UIFormField from "../../../UI/Form/Field";
 import UIFormSelect from "../../../UI/Form/Select";
 import UIFormFieldArray from "../../../UI/Form/FieldArray";
 import UIFormFieldArrayDisplay from "../../../UI/Form/FieldArrayDisplay";
 import UICodedTermItem from "../../../UI/CodedTerm/Item";
 import { useCodeLists } from "@js/context/code-list-context";
+import { isEDTFValid } from "../../../../services/helpers";
```

Comment:
As long as we're making some updates to this commit, let's use the relative import syntax as seen in line 9.  All those `../../` are making my brain hurt:)